### DISCUSSION TITLE
decomposeEssentialMat function missing?

### DISCUSSION THREAD

**Discussion by D3D-s** [emgucv/emgucv#724] [Q&A]
Hello, 

I can't find in EMGU the decomposeEssentialMat function :(

I have Essential Matrix and calibrated cameras now I need to Projection matrix to use the TriangulatePoints function. 
How I can find the Rotation and Translation matrix from the Essential matrix?

**Comment by emgucv** [comment]
It has been added here a few weeks ago:
https://github.com/emgucv/emgucv/blob/EMGUCV_5_0/Emgu.CV/3d/3dInvoke.cs#L585

It will be included in the next release.
